news.yaml

```diff
@@ -1,4 +1,11 @@
 items:
+ - title: Mypy 0.740 released
+   date: 16 October 2019
+   text: |
+         Mypy 0.740 was released.  This release type checking for str.format and improved checking of untyped functions. Read the
+         <a href="http://mypy-lang.blogspot.com/2019/10/mypy-740-released.html">blog post</a>
```

Comment:
The link is broken, blog post has a zero in front: http://mypy-lang.blogspot.com/2019/10/mypy-0740-released.html